src/DssSpell.sol

```diff
@@ -18,9 +18,10 @@ pragma solidity 0.8.16;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
-import { VatAbstract } from "dss-interfaces/dss/VatAbstract.sol";
 
-interface RwaLiquidationOracleLike {
+interface RwaLiquidationLike {
+    function ilks(bytes32) external view returns (string memory, address, uint48, uint48);
+    function init(bytes32, uint256, string calldata, uint48) external;
     function bump(bytes32 ilk, uint256 val) external;
```

Comment:
function `bump` is not used in the spell

Reply:
fixed!